setup/www/resources/config/nodejs.org

```diff
@@ -184,6 +191,14 @@ server {
 }
 
 server {
+    listen *:443 ssl http2;
+    listen [::]:443 ssl http2;
+    server_name newsletter.nodejs.org;
```

Comment:
@rvagg: As always; I goofed. When I refactored how we add SSL certificates I put it in [a partial](https://github.com/nodejs/build/blob/master/setup/www/resources/config/ssl-defaults.conf), so my thought was that we had to include it. What I had forgotten was that we chose to include it as default and override it [where needed](https://github.com/nodejs/build/blame/master/setup/www/resources/config/iojs.org#L13..L15).